File: level1/05. 가운데 글자 가져오기/가운데 글자 가져오기_이상성.js
function solution(s) {
    var answer = '';
    
    if(s.length%2===0){//길이가 짝수일 때
        let cnt = s.length/2-1;
        answer = s.charAt(cnt)+s.charAt(cnt+1)
    } else{//길이가 홀수일 때
        let cnt = Math.floor(s.length/2);
        answer = s.charAt(cnt)
    }
    
    return answer;
}

File: level1/05. 가운데 글자 가져오기/가운데 글자 가져오기_이상성.java
import java.lang.Math;
import java.lang.Integer;

class Solution {
    public String solution(String s) {
        String answer = "";
        
        
        
        if(s.length()%2==0){
            int cnt = s.length()/2-1;
            answer += s.charAt(cnt);
            answer += s.charAt(cnt+1);
        } else {
            int cnt = Integer.parseInt(""+(s.length()/2));
            answer += s.charAt(cnt);
        }
        
        return answer;
    }
}